big.js

```diff
@@ -986,8 +988,8 @@
     /*
      * Return a string representing the value of this Big.
      * Return exponential notation if this Big has a positive exponent equal to
-     * or greater than TO_EXP_POS, or a negative exponent equal to or less than
-     * TO_EXP_NEG.
+     * or greater than Big.TO_EXP_POS, or a negative exponent equal to or less than
+     * Big.TO_EXP_NEG.
      */
     P.toString = P.valueOf = P.toJSON = function () {
```

Comment:
`var Big = x.constructor;` would need to be included before this (see the `format` function for example), otherwise `Big` will refer always to the original `Big` constructor.